frontend/src/__tests__/cypress/cypress/fixtures/e2e/modelRegistry/testRegisterModel.yaml

```
@@ -30,4 +31,4 @@ uriVersion2: "s3://test-bucket/models/tensorflow-model-v2?endpoint=http%3A%2F%2F
 
 newNameSuffix: "-edited"
 newDescription: "Updated description for e2e test"
-  
\ No newline at end of file
+deployProjectNamePrefix: "test-deploy-project"  
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)